### ISSUE TITLE
"unable to open video device" when having 2 cameras

### ISSUE THREAD

**Issue by AntouanK** [ccrisan/motioneye#1941]
Hi there

having a bit of a weird issue.
I have a small laptop running arch and docker in there.
I am running motioneye via a docker container. Nothing weird there, I did that with a raspberry pi for the past 3 months, no problem at all.

I pass all the devices to the container
```
docker run --name="motioneye" \
    -p 8765:8765 \
    --hostname="motioneye" \
    -v /etc/localtime:/etc/localtime:ro \
    -v /etc/motioneye:/etc/motioneye \
    -v /var/lib/motioneye:/var/lib/motioneye \
    --restart="always" \
    --detach=true \
    --device=/dev/video0 \
    --device=/dev/video1 \
    --device=/dev/video2 \
    --device=/dev/video3 \
    ccrisan/motioneye:master-amd64
```
( btw why does each camera create two devices? from what I understand 0 and 1 is the first camera, and 2 and 3 is the second )

Anyway.
What's weird is that now that I switched to the laptop, I have 2 usb cameras ( identical ) and one of them is not working when I switch the resolution high.
Each camera works fine on its own. Goes up to 1920x1080, no problem.

Let's say I add camera X.
Works fine. I change resolution to any acceptable setting, all good.
As soon as I activate the camera Y, I get the grey screen wit the "unable to open video device" message.
If I set camera Y to 320x240, I get image there.
If I set camera X to 320x240, and camera Y to 1920x1080, all good!
As soon as I turn camera X to a higher resolution, again the grey screen to one of them.

To summarize: 
each camera on it's own, works fine on all resolutions. But cannot have both cameras with high resolution, one has to be at 320x240!
What can be the issue? Any ideas?

Thank you.

**Comment by AntouanK** [comment]
duplicate
